app/komponenter/veilederpanel/veilederpanel.module.css

```diff
@@ -8,15 +8,15 @@
   }
 }
 
-.tekstInnholdMellomrom {
+.bunnMargin {
   margin-bottom: 1.2rem;
 }
 
-.poster {
+.posterKonteiner {
```

Comment:
Dette er ikke en kontainer som ligger rundt GuidePanel, men en style som settes på veiledningspanelet. Så et mer beskrivende navn ville vært f.eks. `veilederPanelPoster` eller bare `poster` siden det ligger i `veilederpanel` css-filen.

Trenger man her å gjøre forskjell på poster og ikke poster egentlig? Eneste forskjellen mellom `veilederpanel` og `posterKonteiner` er padding: 0 og width: 80% for mobil.

Reply:
@sarahjelle Ikonet som er til venstre på default guidepanel. Da ønsker man at boksen skal line opp med resten når den tar full bredde og at hele panelet passer på skjermen når man går til mobil.

Reply:
Se commit: 7170681